app/shop.py

```diff
@@ -0,0 +1,33 @@
+import datetime
```

Comment:
The import statement should use `import datetime` as specified in the checklist. Avoid using other forms of import for datetime.